lib/util/git/index.ts

```diff
@@ -704,12 +704,13 @@ export async function prepareCommit({
 }: CommitFilesConfig): Promise<CommitResult | null> {
   const { localDir } = GlobalConfig.get();
   await syncGit();
-  logger.debug(`Preparing files for commiting to branch ${branchName}`);
+  logger.debug(`Preparing files for committing to branch ${branchName}`);
   await handleCommitAuth(localDir);
   try {
     await git.reset(ResetMode.HARD);
     await git.raw(['clean', '-fd']);
     await git.checkout(['-B', branchName, 'origin/' + config.currentBranch]);
+    const oldSha = (await git.revparse([branchName])).trim();
```

Comment:
I think we already have branch SHAs stored in a record

Reply:
Don't we store them after commit and push, though?

Reply:
Never mind, I get it